timesketch/lib/analyzers/chain.py

```diff
@@ -0,0 +1,86 @@
+"""The sketch analyzer for chained events."""
+from __future__ import unicode_literals
+
+import collections
+import uuid
+
+from timesketch.lib import emojis
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+from timesketch.lib.analyzers import chain_plugins  # pylint: disable=unused-import
+from timesketch.lib.analyzers.chain_plugins import manager as chain_manager
+
+
+class ChainSketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for chained events."""
```

Comment:
Can you add a little slurb about what the analyzer do? We have been pretty relaxed on this up until now, but I think it would be great to explain a little in the docstring to make it clear for anyone looking at examples etc.

Reply:
this better?

Reply:
perfect